libraries/teams-scenarios/searchBasedMessagingExtension/src/index.ts

```diff
@@ -7,7 +7,7 @@ import * as restify from 'restify';
 
 // Import required bot services.
 // See https://aka.ms/bot-services to learn more about the different parts of a bot.
-import { BotFrameworkAdapter, MemoryStorage, UserState } from 'botbuilder';
+import { BotFrameworkAdapter, MemoryStorage, UserState } from '../../../botbuilder';
```

Comment:
Do we need this change? from importing `botbuilder` to `../../../botbuilder`

Reply:
I reverted it. Thanks!